static/app/views/starfish/queries/useSpanTransactionMetrics.tsx

```diff
@@ -76,7 +75,6 @@ function getEventView(
         `sum(${SPAN_SELF_TIME})`,
         `avg(${SPAN_SELF_TIME})`,
         'time_spent_percentage()',
-        'transaction.op',
```

Comment:
@gggritso do you know if we need this, I believe this is just code from when we were filtering out non http transactions. If anything by removing it, we avoid the same transaction being displayed multiple times in the table when the transaction has more then one possible ops.

Reply:
That sounds right, but why does this prevent the average size column from showing up?

Reply:
For some reason there are two different grouping here, when `transaction.op` is not there, we have a size. If we don't add the `transaction.op` field, these two values will be combined into one
![image](https://github.com/getsentry/sentry/assets/44422760/bd60711e-ad4e-44d8-af92-a4eeb151991f)

^Not sure if that makes perfect sense here. Ideally this shouldn't happen tho, there's something weird going on with the data causing different ops.

Reply:
@gggritso, what might be happening is we can't collect sizes on pageload, cc @AbhiPrasad, do you think that's possible?

Reply:
Actually that can't be it, why would spm() be 0 there

Reply:
🤔 is it possible that Ingest isn't collecting the `transaction.op` tag for some reason?

Reply:
Yeah i'm starting to think that, going to forward this to them!